src/App.jsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import './App.css';
 
-export const App = () => (
+export const App = ({ a, b }) => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <p>
+      Sum of
+      {' '}
+      {a}
+      {' '}
+      and
+      {' '}
+      {b}
+      {' '}
+      is
+      {' '}
+      {a + b}
+      {' '}
+    </p>
```

Comment:
- you need to move the p tag inside the Sum component
- you need to calculate a + b and show text inside the Sum component as well